moveit_planners/ompl/ompl_interface/include/moveit/ompl_interface/detail/ompl_constraints.h

```diff
@@ -93,6 +93,47 @@ struct Bounds
   inline double derivative(double value) const;
 };
 
+/** \brief Represents upper and lower bound on the elements of a vector.
+ *
+ * OMPL ConstrainedStateSpace requires a model of the constraints given as:
+ *  f1(joint_values) = 0
+ *  f2(joint_values) = 0
+ *  f3(joint_values) = 0
+ *  ...
+ *
+ * So we use a penalty function to convert bounds to an equality constraint.
+ * If you do need equality constraint, you can represent them by setting the upper bound and lower bound almost equal.
+ * Or you can use the EqualityPositionConstraint version by setting the name of the constraint to
+ * "use_equality_constraints". But the latter will ignore bounds on other dimensions.
+ * **/
+class Bounds2
```

Comment:
```suggestion
class Bounds
```

Reply:
Maybe you are still working on this, but I assume that eventually you will rename Bounds2 to Bounds and eliminate the Bounds class above, right?

Reply:
Indeed, I am committing work in progress. Sorry for the confusion, I'll let you know when the refactoring is finished.